utils/_context/_scenarios/profiling.py

```diff
@@ -32,12 +32,6 @@ def configure(self, config: pytest.Config):
             self.weblog_container.environment["LD_PRELOAD"] = (
                 "/opt/datadog/continuousprofiler/Datadog.Linux.ApiWrapper.x64.so"
             )
-        elif library == "python":
```

Comment:
Will it work for backport ? if no, a condition on library version must be added (ping me if you need help)